Test/WhiskeyTest.psm1

```diff
@@ -294,12 +290,9 @@ function New-WhiskeyTestContext
         }
     }
 
-    $context = New-WhiskeyContextObject
+    $context = New-WhiskeyContext -Environment 'Verification' -ConfigurationPath $ConfigurationPath

```

Comment:
Why this change? It requires any test that calls it to setup an entire build environment, which is expensive and makes tests have to know everything about Whiskey instead of just what they care about.

Reply:
Oh this change was originally made when I initially made `TaskDefaults` a global config property that was set in `New-WhiskeyContext`. Now that `TaskDefaults` is a task, this is no longer necessary. I'll revert this.